robyn/__init__.py

```diff
@@ -173,6 +173,16 @@ def start(self, url: str = "127.0.0.1", port: int = 8080):
         logger.info(f"Robyn version: {__version__}")
         logger.info(f"Starting server at {url}:{port}")
 
+        endpoint = {}
+
+        routes = self.router.get_routes()
+
+        for route in routes:
```

Comment:
Hey @carlosm27 :wave: 

Thank you for the PR :smile: 
We should be using something like this

```suggestion
        for method, route in routes:
                logger.info("Logging endpoint: {method: %s, route: %s}", method, route)
```

As the logging library is optimized for deferred string evaluation https://docs.python.org/3/howto/logging.html#optimization

Reply:
Thank you for the reviews @IdoKendo :smile: